packages/renderer/src/lib/service/ServiceActions.spec.ts

```diff
@@ -63,3 +63,12 @@ test('Expect no error and status deleting service', async () => {
   expect(updateMock).toHaveBeenCalled();
   expect(deleteMock).toHaveBeenCalled();
 });
+
+test('Expect buttons to be disabled when deleting', async () => {
+  const { component } = render(ServiceActions, { service });
+  component.status = 'DELETING';
```

Comment:
I thought it was suspicious that the test still worked when the status was set on `component` instead of `service` ;-) and indeed this test is still passing because the button is already in progress from the previous test - if you reorder the tests, it fails.

I didn't have enough time to figure out why the tests aren't independent, but needs some cleanup between. To confirm the test is working it's probably good to confirm that it fails when this line is deleted and not otherwise.